config/api.yml

```diff
@@ -2800,6 +2804,10 @@
         :identifier: catalogitem_delete
       - :name: order
         :identifier: svc_catalog_provision
+      - :name: archive
+        :identifier: svc_catalog_archive
+      - :name: unarchive
+        :identifier: svc_catalog_unarchive
         :options:
         - :validate_action
```

Comment:
reason for the one failure, options: validate_action here is meant for the order action on 2805, so move archive/unarchive below 2812.

Reply:
:+1: thanks